troubleshooting/README.md

```diff
@@ -1,6 +1,46 @@
-Run
+# Troubleshooting
+
+### Log collector script
+
+The log collector script will collect  
+- output of describe pod
+- yaml spec of pod
+- efs-utils logs from /var/log/amazon/efs
```

Comment:
nit : markdown for file path, same as below

Reply:
done